third-party/java/intellij/BUCK

```diff
@@ -48,3 +49,8 @@ prebuilt_jar(
     name = "jdom",
     binary_jar = "jdom.jar",
 )
+
+prebuilt_jar(
+    name = "android",
+    binary_jar = "android.jar",
```

Comment:
I got this JAR from IntelliJ installation. it was needed so that we could find the `FeatureEnableService` symbol